ReactSkia/components/RSkComponentText.cpp

```diff
@@ -65,9 +87,12 @@ void RSkComponentParagraph::OnPaint(SkCanvas *canvas) {
         if(!parent->expectedAttachmentCount || (parent->expectedAttachmentCount == parent->currentAttachmentCount)) {
             auto frame = parent->getAbsoluteFrame();
             paragraph->layout(frame.size.width);
-            SkScalar paraHeight = paragraph->getHeight();
+            paragraph->getLineMetrics(metrics);
 
-            yOffset = yPosOffset(data.attributedString, paraHeight, frame);
+            ellipSize(parent->paraBuilder, metrics, props.paragraphAttributes, frame);
+            auto paragraph = parent->paraBuilder->Build();
```

Comment:
paragraph rebuild and layout is required to be done only if maxlines and ellipsis is set.

Reply:
is auto required again here ? since its already declared.

Reply:
Now, Paragraph rebuild and layout is done based on if maxLines and Ellipsis are set.
Removed auto keyword, as paragraph is already declared.